### ISSUE TITLE
Near constant loss

### ISSUE THREAD

**Issue by nscotto** [octiapp/KerasPersonLab#16]
Does someone have managed to get results with this repository?

My loss is near constant, oscillating around 1.50.
I should mention that I have removed the instance segmentation module, but that shouldn't impact the pose estimation module.

**Comment by AroMorin** [comment]
Have you solved this? Just curious, I haven't worked with this lib before.

**Comment by nscotto** [comment]
@AroMorin 
I had more success with the other repository based on Tensorflow, so I stopped trying with this one.

**Comment by AroMorin** [comment]
I see. The other libraries usually operate with TF 1.X. The advantage of this library is that it uses Keras, so the interface doesn't change much between TF1.X and TF2.X. Also, it uses MIT License which is permissive. Others typically have restrictive licenses or none at all.

**Comment by AroMorin** [comment]
@nscotto can you tell me what other libraries worked for you? Did you measure the accuracy using COCO ?

**Comment by nscotto** [comment]
It's the [Tensorflow Personlab](https://github.com/scnuhealthy/Tensorflow_PersonLab) repository, roughly the equivalent of this one.
I don't remember the mAP value sorry, they provide pre-trained weights in case you want to try.
